_posts/2022-09-22-lxqt-modules.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title: About Modules in LXQt
+slug: about-modules-in-lxqt
+date: 2022-09-20 22:08
+promoted: true
+categories: blog
+---
+
+LXQt is always described as "lightweight and modular" - let's look a little bit closer at the second one.
+This screen is familiar, users can choose which parts to run always on startup, or can stop a component:
+
+![Session Settings of LXQt](../../../../../images/posts/sessionsettings.png)
+
+In a wider sense *modular* includes also the window manager which can be chosen or another default
+filemanager which could be integrated easily too.
+
+The main difference from a simple autostart entry for an application and a LXQt Module is
```

Comment:
The a VS an thing sometimes has me wondering which is better.  If it was just a Module, I would always go with "a Module" here, but when it's something like "an LXQt... Module", I'd go with an... but right now I'm thinking I need to actually research a bit to see which one is more right.  :V

Reply:
Yeah, this an|a thing... I went by feeling, but started to use "an" everywhere...